src/game/shell.c

```diff
@@ -199,21 +296,26 @@ bool Shell_MakeScreenshot()
         break;
     }
 
-    char path[20];
-    for (int i = 0; i < 10000; i++) {
-        sprintf(path, "screenshot%04d.%s", i, ext);
-        char *full_path = NULL;
-        File_GetFullPath(path, &full_path);
+    // Screenshot name
+    char ss_name[LEVEL_TITLE_SIZE + 25]; // 50
```

Comment:
Please extract the magic number to a module-level constant, use more meaningful variable names, and remove verbose comments in favor of variable names